data-plane/receiver/src/main/java/dev/knative/eventing/kafka/broker/receiver/IngressProducer.java

```diff
@@ -45,4 +46,9 @@
    */
   String getTopic();
 
+  /**
+   * @return the resource associated with this producer.
+   */
+  DataPlaneContract.Reference getReference();
```

Comment:
did we do the references on a different PR before?

Reply:
Yes, I referenced the PR in the PR body: https://github.com/knative-sandbox/eventing-kafka-broker/pull/1701